public/js/home.js

```diff
@@ -1268,21 +1268,16 @@ $.ajax("/version").then(ver => $("#version").text(ver));
 $.ajax("/updates").then(upt => {
 	$("#updates").html(upt);
 	document.getElementById("changelog").outerHTML = "<h2 class='info-header'>Version History/What's New:</h2>";
-    const items = document.querySelectorAll("#updates h2");
+    var items = [...document.querySelectorAll("#updates h2")];
```

Comment:
@kdk1616 Why did you change `const` to `var` here? We should always avoid `var` in favor of `const` or `let`.